FWCore/Catalog/interface/FileLocator.h

```diff
@@ -37,14 +36,10 @@ namespace edm {
 
     void init(std::string const& catUrl, bool fallback);
 
-    void parseRule(xercesc::DOMNode* ruleNode,
-                   ProtocolRules& rules);
+    void parseRule(xercesc::DOMNode* ruleNode, ProtocolRules& rules);
 
-    std::string applyRules(ProtocolRules const& protocolRules,
-                           std::string const& protocol,
-                           std::string const& destination,
-                           bool direct,
-                           std::string name) const;
+    std::string applyRules(ProtocolRules const& protocolRules, std::string const& protocol,
+                           std::string const& destination, bool direct, std::string name) const;
```

Comment:
is this a purely automated justification of function arguments?
At least in some cases this will make the code significantly less readable.
-1

Reply:
hi @slava77 - yes, its clang-format with the configuration in the .clang-format file included in the PR.

Reply:
In addition to plain readability, would this also trigger a shift in all lines of arguments if e.g. a new argument is inserted in front?